Libraries/Components/TextInput/TextInput.js

```diff
@@ -411,8 +411,10 @@ var TextInput = React.createClass({
   _renderIOS: function() {
     var textContainer;
 
-    var props = this.props;
-    props.style = [styles.input, this.props.style];
+	  var props = {
+			...this.props,
+		  style: [styles.input, this.props.style]
+    };
```

Comment:
can you update this to use spaces instead of tabs?
also put a trailing comma after the last property